### PR TITLE
[docs] Add a roadmap

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs @miherlosev 

http://strelyaev-w8:8080/testcafe/roadmap/

WIP because we still need to provide meaningful titles and descriptions to all these features on GitHub.

### CODE REVIEW

File: docs/articles/roadmap/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Feature                                                                                           | Status
+------------------------------------------------------------------------------------------------- | -------------------
+[Visual Regression Testing](https://github.com/DevExpress/testcafe/issues/1207)                   | Under Consideration
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Under Consideration
+[Cucumber Integration](https://github.com/DevExpress/testcafe/issues/2835)                        | Under Consideration
+[Selector Debug Panel](https://github.com/DevExpress/testcafe/issues/3244)                        | Under Consideration
+[Entire Page Screenshots](https://github.com/DevExpress/testcafe/issues/1520)                     | Under Consideration
+[Custom Actions](https://github.com/DevExpress/testcafe/issues/1535)                              | Under Consideration
+[Custom TypeScript Compilation Options](https://github.com/DevExpress/testcafe/issues/1845)       | Under Consideration
+[Conditional Browser Skip](https://github.com/DevExpress/testcafe/issues/481)                     | Under Consideration
```
Comment (helen-dikareva) [inline docs/articles/roadmap/README.md]: IMO it's not pretty clear name, what about `Conditional Test Execution` or `Conditional Test Skip` (not sure). We are going to implement a predicate function for skipping some tests, not only depends on browsers.

Comment (miherlosev) [reply docs/articles/roadmap/README.md]: `Conditional Test Execution` is a good feature name

**Comment by VasilyStrelyaev** [comment]
> Also, we decided to add in the roadmap the features that are implemented already - configuration file, video recording and live mode. This should show that we are already implementing the features from the roadmap.

We now say that `here are the features we are working on`. It's clear and understandable. The roadmap is "dynamic" - we add and remove features as we progress.

If we do as you say, we should write something like `Roadmap 2019` - `here are the features planned for the year 2019`. And then, yes, we can have a static list of features and update their statuses.

**Comment by VasilyStrelyaev** [comment]
Otherwise, it will be confusing

File: docs/articles/roadmap/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Under Consideration
+[Cucumber Integration](https://github.com/DevExpress/testcafe/issues/2835)                        | Under Consideration
+[Selector Debug Panel](https://github.com/DevExpress/testcafe/issues/3244)                        | Under Consideration
+[Screenshots of the Entire Page](https://github.com/DevExpress/testcafe/issues/1520)              | Under Consideration
+[Custom Actions](https://github.com/DevExpress/testcafe/issues/1535)                              | Under Consideration
+[Custom TypeScript Compilation Options](https://github.com/DevExpress/testcafe/issues/1845)       | Under Consideration
+[Conditional Test Skip](https://github.com/DevExpress/testcafe/issues/1626)                       | Under Consideration
+[Browser Information Available in Test Code](https://github.com/DevExpress/testcafe/issues/481)   | Under Consideration
+
+> We make no guarantee that we implement all the features listed here. Our decisions depend on your feedback.
```
Comment (kirovboris) [inline docs/articles/roadmap/README.md]: Do we really need this bureaucratic disclaimer?

Comment (miherlosev) [reply docs/articles/roadmap/README.md]: DevExpress Roadmap has the similar [disclaimer](
https://community.devexpress.com/blogs/ctodx/archive/2018/03/08/devexpress-roadmap-for-2018.aspx).

**Comment by kirovboris** [comment]
Also we should structurize the roadmap.

**Comment by VasilyStrelyaev** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
@miherlosev , take a look please!

**Pull request merged**